spec/spec_helper_acceptance.rb

```diff
@@ -27,7 +27,7 @@
   c.before :suite do
     begin
       # Install modules and dependencies from spec/fixtures/modules
-      copy_fixture_modules_to( hosts )
+      # copy_fixture_modules_to( hosts )
```

Comment:
Did you mean to leave this commented out?

Reply:
No, I'll delete the rest of it